torch/onnx/__init__.py

```diff
@@ -310,9 +310,24 @@ def forward(self, x):
                         custom_opsets, enable_onnx_checker, use_external_data_format)
 
 
-def export_to_pretty_string(*args, **kwargs):
+def export_to_pretty_string(*args, **kwargs) -> str:
     r"""
-    Same as :func:`export`, but returns a text representation of the exported model.
+    Similar to :func:`export`, but returns a text representation of the ONNX
+    model. Only differences in args listed below. All other args are the same
```

Comment:
nit: We have some works on deprecating arguments in `export`. Need keep the args in `export` and `export_to_pretty_string` in sync.

Reply:
Thanks for catching that. Done!